### PR TITLE
Fama-French alpha beta

### PR DESCRIPTION
**Pull request by sbussmann**
Adds Fama-French benchmark to alpha and beta plots in Bayesian tear sheet.

Not currently functional.  Line 106:
start = pm.find_MAP(fmin=sp.optimize.fmin_powell)

fails with the following TypeError:
TypeError: Ambiguous name: beta - please check the names of the inputs of your function for duplicates.

I might need some help to debug this one.

### CODE REVIEW

**Comment by sbussmann** [comment]
Fama-French model can be used as a benchmark for alpha beta plots in the Bayesian tear sheet.  I think this code is ready to be implemented on quantopian:master.

File: WHATSNEW.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 These are new features and improvements of note in each release.
 
+## v0.3 (TBA)
```
Comment (twiecki) [inline WHATSNEW.md]: Still some problems with this branch not being up-to-date. Maybe redo this PR off of a clean master, or try rebasing.

File: pyfolio/bayesian.py [hunk 2]
```diff
@@ -69,16 +77,40 @@ def model_returns_t_alpha_beta(data, bmark, samples=2000):
...
-        alpha_reg = pm.Normal('alpha', mu=0, sd=.1, testval=alpha_init)
-        beta_reg = pm.Normal('beta', mu=0, sd=1, testval=beta_init)
-
+        #beta_init, alpha_init = np.linalg.lstsq(
+        X = bmark.loc[data_no_missing.index]
+        X['ones'] = np.ones(len(X))
+        y = data_no_missing
+        alphabeta_init = np.linalg.lstsq(X, y)[0]#[:2]
+
+        #beta_init, alpha_init = sp.stats.linregress(
```
Comment (twiecki) [inline pyfolio/bayesian.py]: Still have to remove commented out code.

File: pyfolio/tears.py [hunk 3]
```diff
@@ -611,23 +643,68 @@ def create_bayesian_tear_sheet(returns, benchmark_rets=None,
...
                                           samples=samples)
+    previous_time = timer("running alpha beta model", previous_time)
 
     # Plot alpha and beta
     row += 1
     ax_alpha = plt.subplot(gs[row, 0])
     ax_beta = plt.subplot(gs[row, 1])
-    sns.distplot((1 + trace_alpha_beta['alpha'][100:])**252 - 1, ax=ax_alpha)
+    if fama_french:
+        sns.distplot((trace_alpha_beta['alpha'][100:]), ax=ax_alpha)
```
Comment (twiecki) [inline pyfolio/tears.py]: Annualize alpha (see above).

**Comment by twiecki** [comment]
Looks great @sbussmann (but see minor comments). The main issue are the git problems so you might have to start from a fresh master with this.

**Comment by sbussmann** [comment]
I brought this branch up to date with quantopian:master and addressed the minor comments.  Hope it's in good shape now!

File: pyfolio/tears.py [hunk 4]
```diff
@@ -589,10 +589,18 @@ def create_bayesian_tear_sheet(returns, benchmark_rets=None,
...
-        benchmark_rets = utils.get_symbol_rets('SPY',
+        benchmark_rets = pd.DataFrame(utils.get_symbol_rets('SPY',
                                                start=returns.index[0],
-                                               end=returns.index[-1])
+                                               end=returns.index[-1]))
+    # unless user indicates otherwise
+    elif benchmark_rets == 'Fama-French':
+        fama_french = True
+        rolling_window = utils.APPROX_BDAYS_PER_MONTH * 6
+        benchmark_rets = timeseries.rolling_fama_french(
```
Comment (twiecki) [inline pyfolio/tears.py]: Instead call https://github.com/quantopian/pyfolio/blob/master/pyfolio/utils.py#L286

**Comment by twiecki** [comment]
Closing in favor of https://github.com/quantopian/pyfolio/pull/200.

**Pull request closed without merging**